cloud/wfm/src/executor/ExecutorRepository.ts

```diff
@@ -0,0 +1,6 @@
+import ProcessInstance from '../process-instance/ProcessInstance';
+interface ExecutorRepository {
+  saveInstance(instance: ProcessInstance): Promise<ProcessInstance>;
```

Comment:
ah, missing the bluebird import here, though it would fall back to the native promise